Dockerfile

```diff
@@ -1,11 +1,12 @@
-FROM        alpine:latest
+FROM alpine:latest
 
-COPY rds_exporter  /bin/
-COPY config.yml           /etc/rds_exporter/config.yml
+COPY rds_exporter /bin/
+COPY config.yml  /etc/rds_exporter/config.yml
 
 RUN apk update && \
     apk add ca-certificates && \
     update-ca-certificates
 
-EXPOSE      9042
-ENTRYPOINT  [ "/bin/rds_exporter", "-config.file=/etc/rds_exporter/config.yml" ]
+EXPOSE 9042
+
+ENTRYPOINT [ "/bin/rds_exporter", "--config.file=/etc/rds_exporter/config.yml" ]
```

Comment:
it's crashed without `--`

Reply:
I've go across the examples of time.Duration unmarshal time in json/yaml, better to use or in string representation. Because you need to cast it into * Seconds anyway, for value in config.yaml. Like this 
```
aws.Int64(aws.TimeUnixMilli(time.Now().UTC().Add(-instance.Interval))),
```
will be crashed because expected Seconds but not `time.Duration`